Sources/PayPalDataCollector/MagnesSDKProtocol.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+import PPRiskMagnes
+
+protocol MagnesSDKProtocol {
+    func setUpWithParams(_ magnesParams: MagnesSetupParams) throws
+    func collectAndSubmit(
+        withPayPalClientMetadataId cmid: String,
+        withAdditionalData additionalData: [String: String]
+    ) throws -> MagnesResult
+}
+
+extension MagnesSDK: MagnesSDKProtocol {
+
+    func setUpWithParams(_ params: MagnesSetupParams) throws {
+        // workaround to avoid lint errors
```

Comment:
What was the linter warning that this threw without the workaround? Just curious.

Reply:
It was the line width greater than 200 or something. And honestly it's because of the Magnes API but it made testing easier as a result.